Model/Session/Storage.php

```diff
@@ -18,7 +18,7 @@ class Storage extends \Magento\Framework\Session\Storage
     public function __construct(
         Share $configShare,
         StoreManagerInterface $storeManager,
-        $namespace = 'yotposmsbump',
```

Comment:
storage namespace shouldn't be related.
need to validate.
if true, lets go with `yotpo_messaging`